core/app/models/spree/store_mailer_logo.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
+  class StoreMailerLogo < Asset
+    if Spree.public_storage_service_name
+      has_one_attached :attachment, service: Spree.public_storage_service_name
+    else
+      has_one_attached :attachment
+    end
+
+    # validates :attachment, content_type: VALID_CONTENT_TYPES
```

Comment:
@aleksandarpetrushev why is this commented out?

Reply:
@damianlegawiec It will not stop from uploading to cloudinary, but will not create the record and show no errors, since it is created after the upload